recipe/recipe.yaml

```diff
@@ -0,0 +1,215 @@
+# numpy will by default use the ABI feature level for the first numpy version
+# that added support for the oldest currently-supported CPython version; see
+# https://github.com/numpy/numpy/blob/v2.0.0rc1/numpy/_core/include/numpy/numpyconfig.h#L124
+
+# there are some tests that cannot really work in emulation, see e.g. numpy/numpy#20445
+# test_new_policy reruns part of test suite; including a CPU feature test that fails in emulation
+# emulation problems (apparently) on aarch
+# https://github.com/numpy/numpy/issues/27045
+# new test failures for 2.2.4; possibly emulation-related; see
+# https://github.com/numpy/numpy/issues/28548
+schema_version: 1
```

Comment:
A lot of these comments need to move to the right place. Some previous comments did also not survive the translations, but should.

Reply:
This should properly arranged and complete now.